stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -75,40 +80,81 @@ protected void onPostExecute(ResponseWrapper result) {
           }
     };
 
-    public Stripe() {
-    }
+    private String defaultPublishableKey;
 
+    /**
+     * A blank constructor to set the key later.
+     */
+    public Stripe() { }
+
+    /**
+     * Constructor with publishable key.
+     *
+     * @param publishableKey the client's publishable key
+     * @throws AuthenticationException if the key is invalid
+     */
     public Stripe(String publishableKey) throws AuthenticationException {
         setDefaultPublishableKey(publishableKey);
     }
 
-    public void setDefaultPublishableKey(String publishableKey) throws AuthenticationException {
+    /**
+     * Call to create a {@link Token}.
+     *
+     * @param card the {@link Card} used for this transaction
+     * @param publishableKey the public key used for this transaction
+     * @param callback a {@link TokenCallback} to receive the result of this operation
+     */
+    public void createToken(
+            final Card card,
+            final String publishableKey,
+            final TokenCallback callback) {
+        createToken(card, publishableKey, null, callback);
+    }
+
+    void setDefaultPublishableKey(String publishableKey) throws AuthenticationException {
         validateKey(publishableKey);
         this.defaultPublishableKey = publishableKey;
     }
 
     private void validateKey(String publishableKey) throws AuthenticationException {
         if (publishableKey == null || publishableKey.length() == 0) {
-            throw new AuthenticationException("Invalid Publishable Key: You must use a valid publishable key to create a token.  For more info, see https://stripe.com/docs/stripe.js.", null, 0);
+            throw new AuthenticationException("Invalid Publishable Key: " +
+                    "You must use a valid publishable key to create a token.  " +
+                    "For more info, see https://stripe.com/docs/stripe.js.", null, 0);
         }
         if (publishableKey.startsWith("sk_")) {
-            throw new AuthenticationException("Invalid Publishable Key: You are using a secret key to create a token, instead of the publishable one. For more info, see https://stripe.com/docs/stripe.js", null, 0);
+            throw new AuthenticationException("Invalid Publishable Key: " +
+                    "You are using a secret key to create a token, " +
+                    "instead of the publishable one. For more info, " +
+                    "see https://stripe.com/docs/stripe.js", null, 0);
         }
     }
 
-    public void requestToken(final String tokenId, final Executor executor, final TokenCallback callback) {
+    void requestToken(final String tokenId, final Executor executor, final TokenCallback callback) {
```

Comment:
These methods should not be used, and were only public because we had tests in the wrong place.